nectar/downloaders/threaded.py

```diff
@@ -56,6 +45,11 @@ def __str__(self):
 class SkipLocation(Exception):
     pass
 
+
+class RetryError(Exception):
```

Comment:
Should this live in the [exceptions module](https://github.com/pulp/nectar/blob/master/nectar/exceptions.py)?